### PR TITLE
Added event type close

### PR DESCRIPTION
**Pull request by SamsamBabadi**
Added event type close
This is a sample that contain close event:
http://tester.advertserve.com/servlet/vast3/zone?zid=33

### CODE REVIEW

**Comment by sokil** [comment]
Hello. Thank you for contributing.

There is already event type `closeLinear` used for linear creatives. As described in specification:

```
/**
      * the user clicked the close button on the creative. The name of this event distinguishes it
      * from the existing “close” event described in the 2008 IAB Digital Video In-Stream Ad Metrics
      * Definitions, which defines the “close” metric as applying to non-linear ads only. The “closeLinear” event
      * extends the “close” event for use in Linear creative.
      */
     const EVENT_TYPE_CLOSELINEAR = 'closeLinear';
```

https://www.iab.com/wp-content/uploads/2015/06/VASTv3_0.pdf (page 33)

As i understand, this is non-standart event type, used by some ad servers? May you show me specification of event type `close` or add comment to your PR that this not standartised type for linear creatives, used by some ad servers?

File: src/Creative/AbstractLinearCreative.php [hunk 1]
```diff
@@ -59,7 +59,10 @@
     
     // the user activated a control to reduce the creative to its original dimensions.
     const EVENT_TYPE_COLLAPSE = 'collapse';
-    
+
+    // the user activated the close control and close the creative.
+    const EVENT_TYPE_CLOSE = 'close';
```
Comment (sokil) [inline src/Creative/AbstractLinearCreative.php:64]: There is already event type closeLinear used for linear creatives.

As i understand, this is non-standart event type, used by some ad servers? May you show me specification of event type close or add comment to your PR that this not standartised type for linear creatives, used by some ad servers?

**Review by sokil — CHANGES_REQUESTED** [review]

**Comment by SamsamBabadi** [comment]
@sokil 
https://www.iab.com/wp-content/uploads/2016/04/VAST4.0_Updated_April_2016.pdf
page 57-58

**Comment by sokil** [comment]
Yes, as i say, `close` event available only for non-linear creatives? but you add it to abstract linear creative. 

Page 57 of specification:

```
NonLinear Ad Metrics

• close: the user clicked or otherwise activated a control for removing the ad, which
fully dispatches the ad from the player environment in a manner that does not allow
the user to re-display the ad.
```

Linear creative may be only skipped. There is also `closeLinear` event, introduced in VAST 3 and not found in VAST 4.

So `close` event in linear creative is invalid by specification. But ad servers use it, so you may add comment to event in to your PR that this not standartised event type for linear creatives, used by some ad servers.

**Comment by sokil** [comment]
Custom element and attribute creation allowed since [0.6](https://github.com/sokil/php-vast/releases/tag/0.6)

**Pull request closed without merging**